index.xhtml

```diff
@@ -708,5 +716,32 @@
       </li>
     </ul>
   </section>
+
+  <section id="rules">
+    <h2>Rules</h2>
+    <p>The discord server, the subreddit and the ##programminglanguages
+    IRC channel follow these rules:</p>
+    <ol>
+      <li>Don't be an asshole:
+        <p>This is not a place for flamewars, personal attacks, insults,
+        slurs, hate speech, and anything else intended to demean or attack.
+        DMs and the block function exist for a reason. Take your fights there,
+        or better yet, don't start them at all. Stay considerate and
+        respectful.</p>
+      </li>
+      <li>Don't derail conversations:
+        <p>Nobody's going to mind if you go slightly off-topic in a channel,
+        but don't derail ongoing conversations, and at least move to the
+        relevant channel if you find that the conversation has gone
+        off-`topic. This is a loosely-defined "don't prevent meaningful
+        discussions with off-topic nonsense", and it is up to interpretation
+        by the moderators.</p>
```

Comment:
s/moderator/operator

Reply:
Yes and no.  While IRC traditionally has operators (people who set the topic, and prevent spam) there, it does not traditionally have "moderators" in the sense that they help to keep it a friendly, constructive, inclusive place. It might be that they both have the +o status but it's relevant what they _do_ with that +o.